include/odp/api/spec/crypto.h

```diff
@@ -76,12 +76,25 @@ typedef enum {
 	/** Triple DES with cipher block chaining */
 	ODP_CIPHER_ALG_3DES_CBC,
 
+	/** Triple DES with Electronic Codebook */
+	ODP_CIPHER_ALG_3DES_ECB,
+
 	/** AES with cipher block chaining */
 	ODP_CIPHER_ALG_AES_CBC,
 
 	/** AES with counter mode */
 	ODP_CIPHER_ALG_AES_CTR,
 
+	/** AES with electronic codebook */
+	ODP_CIPHER_ALG_AES_ECB,
+
+	/** AES with 128-bit cipher feedback */
+	ODP_CIPHER_ALG_AES_CFB128,
```

Comment:
The shift size could be added into session parameters (like digest_len and aad_len). This would avoid later enum additions for different shift sizes. How likely it is that e.g. 1 or 8 bit shift needs to be supported, or other chiphers would be used in CFB mode?

E.g. openSSL seems to have 1, 8 and 128 bit shift sizes supported.

EVP_aes_128_cfb(), EVP_aes_192_cfb(), EVP_aes_256_cfb(), EVP_aes_128_cfb1(), EVP_aes_192_cfb1(), EVP_aes_256_cfb1(), EVP_aes_128_cfb8(), EVP_aes_192_cfb8(), EVP_aes_256_cfb8(), EVP_aes_128_cfb128(), EVP_aes_192_cfb128(), EVP_aes_256_cfb128()

Reply:
Shift size is quite unique to cfb mode. It takes only several fixed values. So I do not think that it makes sense to move it to session params.

Reply:
OK.